docs/current_docs/index.mdx

```diff
@@ -79,4 +79,4 @@ Dagger Cloud also collects telemetry from all your organization's Dagger Engines
 
 ### The Daggerverse
 
-The [Daggerverse](https://daggerverse.dev) is a free service run by Dagger, which indexes all publicly available Dagger Modules, and lets you easily search and consume them.
+The [Daggerverse](https://daggerverse.dev) is a free service run by Dagger, which indexes all publicly available Dagger modules, and lets you easily search and consume them.
```

Comment:
this usage makes sense since the Daggerverse indexes many specific implementations of software components, Dagger modules, that are built using the Dagger Modules feature.